### PR TITLE
spec: convert debugger spec to expect

### PR DESCRIPTION
**Pull request by codebytere**
Ongoing work to migrate our specs from `assert` to chai's `expect` for better error messages.

This PR updates the `debugger` spec.

### CODE REVIEW

File: spec/api-debugger-spec.js [hunk 1]
```diff
@@ -78,57 +83,62 @@ describe('debugger module', () => {
       }
     })
 
-    it('retuns response', (done) => {
+    it('returns response', done => {
```
Comment (alexeykuzmin) [inline spec/api-debugger-spec.js]: 👍

File: spec/api-debugger-spec.js [hunk 2]
```diff
@@ -78,57 +83,62 @@ describe('debugger module', () => {
...
-        return done('unexpected error : ' + err)
+        return done(`unexpected error : ${err}`)
       }
-      var callback = function (err, res) {
-        assert(!err.message)
-        assert(!res.wasThrown)
-        assert.equal(res.result.value, 6)
+
+      const callback = (err, res) => {
+        expect(err.message).to.not.exist()
```
Comment (alexeykuzmin) [inline spec/api-debugger-spec.js]: There shouldn't be a `message` property on `err`?
Or it should be there but have a value of `null` or `undefined`?

File: spec/api-debugger-spec.js [hunk 3]
```diff
@@ -78,57 +83,62 @@ describe('debugger module', () => {
...
+        expect(res.wasThrown).to.be.undefined()
```
Comment (alexeykuzmin) [inline spec/api-debugger-spec.js]: The same question as for the `message` above.

File: spec/api-debugger-spec.js (hunk 2 above)
Comment (codebytere) [reply spec/api-debugger-spec.js]: it should be undefined :)

File: spec/api-debugger-spec.js (hunk 3 above)
Comment (codebytere) [reply spec/api-debugger-spec.js]: this should also be undefined

**Pull request merged**